actions/src/main/kotlin/com/alfresco/content/actions/ContextualActionsViewModel.kt

```diff
@@ -152,10 +153,14 @@ class ContextualActionsViewModel(
     private fun sharedActions(entry: Entry, entries: List<Entry>): List<Action> {
         val actions = mutableListOf<Action>()
         // Added Favorite Action
-        if (entries.any { !it.isFavorite }) {
-            actions.add(ActionAddFavorite(entry, entries))
-        } else {
-            actions.add(ActionRemoveFavorite(entry, entries))
+        val version = SearchRepository().getPrefsServerVersion()
+
+        if (version.toInt() >= 23) {
```

Comment:
Can we store version number 23 into a variable and then use it here?

Reply:
Its using single time so global variable may create extra space in memory for this

Reply:
done